cpp/src/memory/columnar_batch.h

```diff
@@ -17,9 +17,9 @@
 
 #include <memory>
 
-#include "arrow/c/bridge.h"
 #include "arrow/c/helpers.h"
 #include "arrow/record_batch.h"
+#include "releases/include/arrow/c/bridge.h"
```

Comment:
Would `#include "arrow/c/bridge.h"` work if adding `release/include` as include directory?

Reply:
no, the wrong file will be included, that's the problem

Reply:
Actually this is to distinguish between `arrow/c/Bridge.h"` and `arrow/c/bridge.h"` by using different prefixes for them, is that right?

Reply:
I used `g++ -MM source.cc...` to get all header files included by a source file, and the result shows that the wrong header file is included.

the compiler will search header files using the include-directories, it will search in the order of `gcc -i dir1 -i dir2`, if the header file `bridge.h` is found in the directory of dir1, the searching will stop.

so, in order to find the right `bridge.h`, we should use the specified path to pick the target `bridge.h`